### PR TITLE
Issue #93: Improve tests coverage for Twitter Send

### PR DESCRIPTION
**Pull request by oneils**
- added test for covering Send functionality

### CODE REVIEW

File: app/main.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"github.com/ChimeraCoder/anaconda"
```
Comment (umputun) [inline app/main.go:7]: nit: pls use goimport os just sort/group imports manually

**Review by umputun — COMMENTED** [review]
looking good, thx. a minor thingy only about imports order

File: app/main.go (hunk 1 above)
Comment (oneils) [reply app/main.go]: done

File: app/proc/twitter.go [hunk 2]
```diff
@@ -12,10 +12,18 @@ import (
 	"github.com/umputun/feed-master/app/feed"
 )
 
-// TwitterClient implements basic publisher of rss itom to twitter
+//go:generate moq -out mocks/tweet_poster.go -pkg mocks -skip-ensure -fmt goimports . TweetPoster
+
+// TweetPoster is the interface for sending messages to telegram
```
Comment (umputun) [inline app/proc/twitter.go:17]: nit: I think this is a typo, i.e. s/telegram/twitter

Comment (oneils) [reply app/proc/twitter.go]: Fixed the comment: It posts Tweets to Twitter, not Telegram.

Does the name `TweetPoster` look good ?

Comment (umputun) [reply app/proc/twitter.go]: yep, TweetPoster makes sense

**Pull request merged**